lib/ui_constants.dart

```diff
@@ -15,4 +15,5 @@ abstract class MovieDetailsUiConstants {
   static const double overviewPadding = 150.0;
   static const double borderRadius = 30;
   static const String releaseDateTextConstant = 'Available on';
+  static const String imagesRouteBeginning = 'https://image.tmdb.org/t/p/w500/';
```

Comment:
Just to follow the convention, I'd rename this as `imageBaseUrl`, but it's your choice!

Reply:
done